Makefile

```diff
@@ -193,9 +194,9 @@ CROSSPLANE_NAMESPACE = upbound-system
 #   aws_secret_access_key = REDACTED'
 #   The associated `ProviderConfig`s will be named as `default` and `peer`.
 # - UPTEST_DATASOURCE_PATH (optional), please see https://github.com/crossplane/uptest#injecting-dynamic-values-and-datasource
-uptest: $(UPTEST) $(KUBECTL) $(KUTTL)
+uptest: $(UPTEST) $(KUBECTL) $(CHAINSAW)
```

Comment:
Here we also need to add `$(CROSSPLANE_CLI)` since uptest forks crossplane cli binary for watching resources. Please see example integration: https://github.com/crossplane-contrib/provider-upjet-aws/blob/3d80d7de7ad7e351f7fcd542809cfe4f66d7fd9a/Makefile#L232.

Without this, the pipeline will not fail, but users will not see the logs for the resources, or can't follow the resources' state during the tests.